js/src/common/utils/humanTime.js

```diff
@@ -6,8 +6,8 @@
  * @return {String}
  */
 export default function humanTime(time) {
-  let m = moment(time);
-  const now = moment();
+  let m = dayjs(time);
```

Comment:
Hmm actually one super quick nitpick: perhaps this should be called `d` instead of `m`?

Reply:
It really doesn't matter. It can be changed.

Reply:
m could be considered a backwards compatible thing I think (if extensions use it), but I agree that it could be confusing long term.